pivomatic

```diff
@@ -70,7 +70,19 @@ check-slug() {
   if [ -z "$SLUG" ]; then
     echo "Here's a list of available products:"
     pivnet products
-    echo "Please choose one and provide it using the -p flag, followed by the release number you wish to deploy."
+    echo "Please choose one and provide the id:"
+    read SLUGID
+    SLUG=$(pivnet products --format json | jq ".[] | select(.id == ${SLUGID}).slug")
+    echo "Chose $SLUG"
+  fi
+
+  if [ -z "$RELEASE" ]; then
+    echo "Here's a list of available releases:"
+    pivnet releases -p $SLUG
+    echo "Please choose one and provide the id:"
+    read RELEASEID
+    RELEASE=$(pivnet releases -p $SLUG --format json | jq ".[] | select(.id == ${RELEASEID}).version")
+    echo "Chose $RELEASE"
     exit 1
```

Comment:
You should remove this line, it kills the process when you provided the release 😉